superset/translations/pt_BR/LC_MESSAGES/messages.po

```diff
@@ -1024,7 +1024,7 @@ msgstr "Salvar como:"
 #: superset/assets/javascripts/dashboard/components/SaveModal.jsx:135
 #: superset/assets/javascripts/explore/components/SaveModal.jsx:210
 msgid "[dashboard name]"
-msgstr "[Nome do painel]"
+msgstr "[dashboard name]"
```

Comment:
Why keep this one in english?

Reply:
Excellent idea to keep it in English! I agree. Sounds good in portuguese too.

Reply:
I think "slice" sounds good in portuguese too because it is a special "feature", right?. What do you think to keep all "pedaço" and "fatia" in English?